toolkit/tools/internal/rpm/rpm.go

```diff
@@ -536,7 +536,7 @@ func buildAllSpecsList(baseDir string) (specPaths []string, err error) {
 
 	specPaths, err = filepath.Glob(specFilesGlob)
 	if err != nil {
-		logger.Log.Errorf("Failed while trying to enumerate all spec files with (%s). Error: %v.", specFilesGlob, err)
+		return nil, fmt.Errorf("Failed while trying to enumerate all spec files with (%s). Error: %w", specFilesGlob, err)
```

Comment:
Please follow [the Go's formatting guidelines for errors](https://staticcheck.dev/docs/checks#ST1005).
```suggestion
		return nil, fmt.Errorf("failed while trying to enumerate all spec files with (%s). Error: %w", specFilesGlob, err)
```